### DISCUSSION TITLE
Welcome to Hacktoberfest-Projects Discussions!

### DISCUSSION THREAD

**Discussion by soumikbaithalu** [Tenet-Coding/Hacktoberfest-Projects#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Harshil-Jani** [comment]
@Everyone HacktoberFest is Up !!! :tada: Let's dive in OpenSource.
Now your PRs will be checked and Merged soon ! :partying_face:
